relay-server/src/actors/processor.rs

```diff
@@ -1070,6 +1070,51 @@ impl EnvelopeProcessorService {
         })
     }
 
+    /// Count the number of profiles that are in the envelope and emit accepted outcome.
+    ///
+    /// "processed" profiles are an abstract data category that does not represent actual data
+    /// going through our pipeline. Instead, the number of accepted "processed" profiles is counted as
+    ///
+    ///     processed_profiles = indexed_profiles + sampled_profiles
+    ///
+    /// The "processed" outcome for sampled profiles is generated by the Kafka producer
+    /// (see `transform_outcome` in [`crate::actors::store`]), but for "indexed" profiles, we count
+    /// the corresponding number of processed profiles here.
+    ///
+    /// NOTE: Instead of emitting a [processed](`DataCategory::Profile`) outcome here,
+    /// we could also do it in sentry, in the same place where the [indexed](`DataCategory::ProfileIndexed`)
+    /// outcome is logged. We do it here to be consistent with profiles that are dropped by dynamic sampling,
+    /// which also count as "processed" even though they did not pass through the `process_profiles` step yet.
+    ///
+    ///
+    /// In the future, we might actually extract metrics from profiles before dynamic sampling,
+    /// like we do for transactions. At that point, this code should be removed, and we should
+    /// enforce rate limits and emit outcomes based on the collect profile metric, as we do for
```

Comment:
The sentence ends here kind of leaving you hanging at the end.
```suggestion
    /// enforce rate limits and emit outcomes based on the collect profile metric, as we do for transactions
```

Reply:
Thanks!